src/commands/folding.rs

```diff
@@ -0,0 +1,70 @@
+/*
+ *   Copyright (c) 2020 Owen Salter <owen@devosmium.xyz>
+ *   All rights reserved.
+ */
+
+use crate::prelude::*;
+extern crate reqwest;
+use std::io::Read;
+use serenity::{
+    http::GuildPagination,
+    model::{
+        guild::PartialGuild,
+        id::GuildId,
+        invite::{Invite, RichInvite},
+    },
+};
+
+#[command]
+#[description = "Provides F@H Leaderboard for the DSC F@H Team."]
+pub fn fahleaderboard(ctx: &mut Context, msg: &Message) -> CommandResult {
+    msg.channel_id.send_message(&ctx.http, |m| {
+        let mut res = reqwest::get("https://api.foldingathome.org/team/262889/members")?;
+        let mut body = String::new();
+        res.read_to_string(&mut body)?;
+        m.embed(|e| {
+            e.title("DSC F@H Stats (Points)");
+            
+            for token in body.split("\"],[\""){
+                let tokens:Vec<&str>= token.split("\",\"").collect();
+                    println!("firstName is {}",tokens[0]);
```

Comment:
make these `debug!` calls so they're logged properly and fitlered by the `RUST_LOG` environment variable